src/index.js

```diff
@@ -0,0 +1,69 @@
+/*eslint-disable*/
+import _ from 'lodash';
+/* eslint-enable */
```

Comment:
- [ ] Kindly remove all `eslint` disable/enable statements in your index.js file. Linters help us write codes in a clean and DRY manner. They also ensure we adopt best practices when writing our code. In this case, you do not need to import `_` from `'lodash'` because it is not needed in your application. Remember, `Lodash` is a library for web development. It has many useful functions that make working with arrays, strings, e.t.c. easier and faster. 👍